pkg/operator/controller/nodeobservability/daemonset.go

```diff
@@ -115,14 +110,11 @@ func (r *NodeObservabilityReconciler) desiredDaemonSet(nodeObs *v1alpha1.NodeObs
 							Command:         []string{"node-observability-agent"},
 							Args: []string{
 								"--tokenFile=/var/run/secrets/kubernetes.io/serviceaccount/token",
-								"--storage=/run",
+								"--storage=/tmp",
```

Comment:
Blocked by image change in node-observability-agent, right?

Reply:
yes